eng/scripts/InstallationCheck.ps1

```diff
@@ -6,27 +6,39 @@ param (
     [string] $Artifact,
 
     [Parameter()]
-    [string] $PipelineWorkspace
+    [string] $PipelineWorkspace,
```

Comment:
Doesn't look like we need or use the PipelineWorkspace variable any longer so we should remove it.